test/alternator/test_manual_requests.py

```diff
@@ -174,6 +175,20 @@ def test_tags_return_empty_body(dynamodb, test_table):
     response = requests.post(req.url, headers=req.headers, data=req.body, verify=False)
     assert not response.text
 
+# Reproduces issue #16233
+def test_create_table_without_keyschema(dynamodb, test_table_s):
+    payload = '{"TableName": "alternator_Test1", "AttributeDefinitions": [{"AttributeName": "lock_key", "AttributeType": "S"}, {"AttributeName": "sort_key", "AttributeType": "S"}], "BillingMode": "PAY_PER_REQUEST"}'
+    req = get_signed_request(dynamodb, 'CreateTable', payload)
+    response = requests.post(req.url, headers=req.headers, data=req.body, verify=False)
+    assert "ValidationException" in response.text and "Missing KeySchema member" in response.text
```

Comment:
As I noted earlier, and I still need to insist on, this test fails when run against DynamoDB, which we don't want (tests need to pass on both Alternator and DynamoDB, to prove the test is correct). You can run this test against DynamoDB with the command
```
test/alternator/run --aws test_manual_requests.py::test_create_table_without_keyschema
```
I just ran it myself, and as I had explained earlier, it failed because DynamoDB's error message in this case is different from Alternator - instead of Alternator's `Missing KeySchema member` message, DynamoDB's error message reads `"1 validation error detected: Value null at \'keySchema\' failed to satisfy constraint: Member must not be null"`
As I noted earlier it's fine that our messages are different as long as the type (ValidationException) is the same. You just need the test to accept both. For example
```python
assert  "ValidationException" in response.text and ("Missing KeySchema member" in response.text or "must not be null" in response.text)
```
As I explained already, the purpose of checking response.text in the test is not to "enshrine" the text's message, but just to confirm that the test itself isn't broken: We had in the past tests which just checked for ValidationException, but a typo (!) caused us to have some other unrelated ValidationException, and the test wasn't actually testing what we thought we were testing.